### PR TITLE
QA for display

### PR DESCRIPTION
**Pull request by larrylawl**
1. Added tests, logging, and assertions for `display`
2. Included our awesome logo in readme and right panel
3. Fixed help link
4. Fixed bug of right panel not updating
5. Larry's PPP
6. Added codacy and netlify badges 
7. Fixed URL naming of website page. (TODO: Fix in github page when this PR is merged in)

### CODE REVIEW

**Comment by larrylawl** [comment]
Fixes #175 #154 #140 #139

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -28,6 +32,7 @@
 public class MainWindow extends UiPart<Stage> {
 
     private static final String FXML = "MainWindow.fxml";
+    private static final String LOGO_URL = "/images/InsurelyticsLogo.png";
```
Comment (chowyiyin) [inline src/main/java/seedu/address/ui/MainWindow.java]: Seems like the logo doesn't appear on the right panel when I run it.

File: docs/team/larrylawl.adoc [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+image::InsurelyticsLogo.png[width="250"]
+endif::[]
+
+ifdef::env-github[]
+image::InsurelyticsLogo.png[width="250"]
+endif::[]
+== Overview
+Insurelytics is a desktop client management application for insurance agents.
+Insurelytics facilitates the tracking of client and policies, and also generate statistical analytics to provide insights for the agent.
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```
Comment (chowyiyin) [inline docs/team/larrylawl.adoc]: I checked haha we have about 80kLoC

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (larrylawl) [reply src/main/java/seedu/address/ui/MainWindow.java]: Can u chk if u have the file in the local machine? 
main/src/main/resources/images/InsurelyticsLogo.png

File: docs/team/larrylawl.adoc (hunk 2 above)
Comment (larrylawl) [reply docs/team/larrylawl.adoc]: Fixed

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (chowyiyin) [reply src/main/java/seedu/address/ui/MainWindow.java]: I think I do. Does anyone else have this problem? @chaitanyabaranwal @ybchen97 @olivercheok20

Comment (larrylawl) [reply src/main/java/seedu/address/ui/MainWindow.java]: works on @ybchen97 tho

Comment (olivercheok20) [reply src/main/java/seedu/address/ui/MainWindow.java]: Works for me as well

**Pull request merged**